docker-image/configure-tool-cache.sh

```diff
@@ -30,14 +30,14 @@ create_symlink() {
   ln -s "$1" "$2"
 }
 
-# --- Python 3.7 ---
+# --- Python 3.10  ---
```

Comment:
Please keep Python 3.7 structure - we would like to have both versions and chose one of them in CI pipelines using this task: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/use-python-version?view=azure-devops